nlp-plugins/docs/NLPTransform-transform.md

```diff
@@ -0,0 +1,210 @@
+# Google NLP Transform
+
+Description
+-----------
+Transforms input text into an information provided by Google Natural Language API.
+The information includes syntax, sentiment, entities, classification of the text data.
+
+Credentials
+-----------
+If the plugin is run on a Google Cloud Dataproc cluster, the service account key does not need to be
+provided and can be set to 'auto-detect'.
+Credentials will be automatically read from the cluster environment.
+
+If the plugin is not run on a Dataproc cluster, the path to a service account key must be provided.
+The service account key can be found on the Dashboard in the Cloud Platform Console.
+Make sure the account key has permission to access Natural Language API.
+The service account key file needs to be available on every node in your cluster and
+must be readable by all users running the job.
+
+Properties
+----------
+**Source Field:** Field which contains an input text.
+
+**Method Name**: Name of Google Natural Language API Method.
```

Comment:
I think it's better to have these all as different transforms rather than have a single transform for them all.

They all do different things with different output schema.